service/src/main/kotlin/fi/espoo/evaka/incomestatement/IncomeStatementControllerCitizen.kt

```diff
@@ -429,7 +439,55 @@ class IncomeStatementControllerCitizen(private val accessControl: AccessControl)
             }
     }
 
-    private fun verifyIncomeStatementModificationsAllowed(
+    private fun verifyIncomeStatementUpdateAllowed(
+        original: IncomeStatement,
+        update: IncomeStatementBody,
+    ) {
+        if (original.status == IncomeStatementStatus.HANDLED) {
```

Comment:
Minusta tämä olisi selkeämpi jos olisi early return draft tilalle
```suggestion
        if (original.status == IncomeStatementStatus.DRAFT) {
            return // Allow all modifications for drafts
        }
        if (original.status == IncomeStatementStatus.HANDLED) {
```
ja sitten alta voisi poistaa iffin
`if (original.status == IncomeStatementStatus.SENT) {`

Reply:
done 👍